prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -161,6 +166,10 @@ VirtualHost "{{ $XMPP_DOMAIN }}"
     {{ end }}
     {{ end }}
 
+    {{ if $PROSODY_RESERVATION_ENABLED }}
+    reservations_api_prefix = { "{{ $PROSODY_RESERVATION_REST_BASE_URL }}" }  
```

Comment:
Should the curly braces around the var be there? That var is expected to be a string, not a table.

Should probably be:
```
reservations_api_prefix = "{{ $PROSODY_RESERVATION_REST_BASE_URL }}" 
```

Reply:
Noted. Will fix it